src/cart/SConscript

```diff
@@ -22,6 +22,25 @@ SRC = ['crt_bulk.c', 'crt_context.c', 'crt_corpc.c',
        'crt_swim.c', 'crt_tree.c', 'crt_tree_flat.c', 'crt_tree_kary.c',
        'crt_tree_knomial.c', 'crt_hlc.c', 'crt_hlct.c']
 
+LIB_SRC = ['crt_utils.c']
+
+##############################
+# Create cart utility library
+##############################
+def build_utility_library():
+    """Build cart utility library"""
+
+    Import('env', 'prereqs')
+    denv = env.Clone()
+    denv.AppendUnique(CPPPATH=[Dir('.').srcnode()])
+    denv.AppendUnique(CPPPATH=[Dir('include').srcnode()])
+
+    prereqs.require(denv, 'mercury', headers_only=True)
```

Comment:
Do you actually need this line (particularly if you remove the dependence on crt_internal.h ?

Reply:
I cannot remove crt_internal.h without re-working /utils/ctl and /utils/self_test.  At this point in time, this has to remain as it is.